### PR TITLE
Add main branch reference instead of master

### PR DESCRIPTION
**Pull request by kashifest**
BMO's default branch is now `main`. This PR removes/updates any reference to `master` branch previously.

### CODE REVIEW

**Comment by kashifest** [comment]
/hold
Holding for a while. Changing the default branch reference to `main` from `master`. Once the transition is done I will unhold it.

**Comment by kashifest** [comment]
/hold cancel
The switch is done. BMO's default branch is now `main`

**Review by fmuyassarov — COMMENTED** [review]
/lgtm

**Comment by kashifest** [comment]
@zaneb @s3rj1k @andfasano @dtantsur can someone please review this PR and approve if it lgty?

File: docs/dev-setup.md [hunk 1]
```diff
@@ -233,18 +233,18 @@ The output can be passed directly to `kubectl apply` like this:
 go run cmd/make-virt-host/main.go openshift_worker_1 | kubectl apply -f -
 ```
 
-When the host is a *master*, include the `-consumer` and
+When the host is a *controlplane*, include the `-consumer` and
```
Comment (zaneb) [inline docs/dev-setup.md:236]: nit: I would say "a controlplane host" or "part of the control plane".

File: pkg/provisioner/ironic/ironic.go [hunk 2]
```diff
@@ -507,7 +507,7 @@ func (p *ironicProvisioner) ValidateManagementAccess(data provisioner.Management
 		return
 
 	case nodes.Active:
-		// The host is already running, maybe it's a master?
+		// The host is already running, maybe it's a controlplane?
```
Comment (zaneb) [inline pkg/provisioner/ironic/ironic.go:510]: ditto

**Review by zaneb — COMMENTED** [review]
The commit message is not especially accurate, since most of these changes are not related to the branch name.

File: docs/dev-setup.md (hunk 1 above)
Comment (kashifest) [reply docs/dev-setup.md]: Done

File: pkg/provisioner/ironic/ironic.go (hunk 2 above)
Comment (kashifest) [reply pkg/provisioner/ironic/ironic.go]: done

**Comment by kashifest** [comment]
> The commit message is not especially accurate, since most of these changes are not related to the branch name.

Updated. PTAL again.

**Review by furkatgofurov7 — COMMENTED** [review]
LGTM

**Review by fmuyassarov — COMMENTED** [review]
/lgtm

**Pull request merged**